graph/src/io/VersionFourMCGReader.cpp

```diff
@@ -132,13 +132,19 @@ std::unique_ptr<metacg::Callgraph> metacg::io::VersionFourMetaCGReader::read() {
 
   auto cg = std::make_unique<Callgraph>();
 
+  auto& jNodes = jsonCG["nodes"];
```

Comment:
This makes "this version of file format v4" incompatible with "the prior version of file format v4".